src/SDKs/Resource/Management.ResourceManager/Microsoft.Azure.Management.ResourceManager.csproj

```diff
@@ -18,4 +18,15 @@
 	<!-- Please do not move/edit code below this line -->
 	<Import Condition="Exists('[MSBuild]::GetPathOfFileAbove(AzSdk.RP.props)')" Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.RP.props'))" />
 	<!-- Please do not move/edit code above this line -->
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
```

Comment:
@wilcobmsft this file was merged incorrectly. Can you check with the original version.